### PR TITLE
Standardize on UI terminology

### PR DESCRIPTION
**Pull request by jeberl**
Summary
---------

No functional changes just wording and description changes

Test Plan
---------
Screen Shot 2021-06-24 at 4 18 45 PM
Screen Shot 2021-06-24 at 4 19 09 PM
Screen Shot 2021-06-24 at 4 19 27 PM
Screen Shot 2021-06-24 at 4 19 39 PM

### CODE REVIEW

**Comment by Dcallies** [comment]
Aren't the internal thing called labels? 

Can you link some screenshots of what things look like now?

**Comment by jeberl** [comment]
@Dcallies Added screenshots to test plan

File: hasher-matcher-actioner/webapp/src/pages/settings/ActionSettingsTab.jsx [hunk 1]
```diff
@@ -123,7 +123,14 @@ export default function ActionSettingsTab() {
       <Card>
         <Card.Header>
           <h2 className="mt-2">Action Definitions</h2>
-          <h5 className="mt-5">Define what to do for different Actions</h5>
+          <h5 className="mt-5">
```
Comment (eriksendc) [inline hasher-matcher-actioner/webapp/src/pages/settings/ActionSettingsTab.jsx:126]: The length of this text is a bit long for a heading (even an h5). Maybe consider `<p>` instead?

Comment (eriksendc) [reply hasher-matcher-actioner/webapp/src/pages/settings/ActionSettingsTab.jsx:126]: Also consider shortening the text. There wasn't an example given for the ActionRules description, for example.

**Review by eriksendc — CHANGES_REQUESTED** [review]
Looks great. Minor comment. Let me know when I can re-review.

File: hasher-matcher-actioner/webapp/src/pages/settings/ActionSettingsTab.jsx (hunk 1 above)
Comment (jeberl) [reply hasher-matcher-actioner/webapp/src/pages/settings/ActionSettingsTab.jsx:126]: I think its better to be verbose at this stage of the project. Too much room for confusion. Will convert to `<p>` though

**Review by eriksendc — APPROVED** [review]

**Pull request merged**